examples/examples.cjsx

```diff
@@ -80,7 +82,9 @@ module.exports = React.createClass
         strokeColor='green'
         strokeWidth='5px'
         interpolate='none'
-        circleDiameter=10 />
+        circleDiameter=10
+        viewBox="0 0 200 40"
```

Comment:
viewBox shouldn't be a prop here correct since you're dynamically generating it from the width/height props?